railties/lib/rails/application/bootstrap.rb

```diff
@@ -35,7 +35,8 @@ module Bootstrap
       # Initialize the logger early in the stack in case we need to log some deprecation.
       initializer :initialize_logger, group: :all do
         Rails.logger ||= config.logger || begin
-          logger = ActiveSupport::Logger.new(config.default_log_file)
+          shift_age = config.log_file_size ? 1 : 0
+          logger = ActiveSupport::Logger.new(config.default_log_file, shift_age, config.log_file_size)
```

Comment:
```suggestion
          logger = if config.log_file_size
            ActiveSupport::Logger.new(config.default_log_file, 0, config.log_file_size)
          else
            ActiveSupport::Logger.new(config.default_log_file)
          end
```

I think the above would read a bit better.

Reply:
sounds good, since `0` will not activate the log rotate, I'll keep it as `1`

Reply:
Nevermind, you're right. I must have messed my testing. This means we'll have another file alongside it, but I guess it's ok.